src/postgres/commands/createPostgresServer/steps/PostgresServerCredUserStep.ts

```diff
@@ -41,7 +41,7 @@ async function validateUser(username: string): Promise<string | undefined> {
         return localize('usernameBeginningMatch', 'The name cannot start with a number.');
     } else if (username.startsWith('pg_')) {
```

Comment:
What about this check - do we need to do `toLowerCase` to prevent people from starting with "PG_"?

Reply:
That's true, "PG_" needs to be lowercase as well. Both versions are considered to be part of the restricted group.